src/components/ProjectAddSummary/component/Administration/LicenseInfoHeader.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+'use client'
+
+import { OverlayTrigger, Tooltip } from 'react-bootstrap'
+import { BiInfoCircle } from 'react-icons/bi'
+import { useTranslations } from 'next-intl'
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+
+const ShowInfoOnHover = ({ text }: { text: string }) => {
+    return (
+        <>
+            <OverlayTrigger overlay={<Tooltip>{text}</Tooltip>}>
+                <span className='d-inline-block'>
+                    <BiInfoCircle />
+                </span>
+            </OverlayTrigger>
+        </>
+    )
+}
```

Comment:
Can this specific one become a global component ?

Reply:
You are duplicating it in more than two sources

Reply:
Made a global component for ShowInfoOnHover & resolved in all occurrences.

Reply:
Made a global component for ShowInfoOnHover & resolved in all occurrences.